corehq/apps/userreports/reports/view.py

```diff
@@ -571,6 +587,80 @@ def get_ajax(self, params):
             json_response['total_row'] = total_row
         return self.render_json_response(json_response)
 
+    def _get_ajax_bootstrap5(self, params):
+        try:
+            return self._render_datatable_page_bootstrap5(params)
+
+        except UserReportsError as err:
+            if settings.DEBUG:
+                raise
+            return self.render_json_response({
+                'draw': int(params.get('draw', 1)),
+                'recordsTotal': 0,
+                'recordsFiltered': 0,
+                'data': [],
+                'error': str(err),
+            })
+
+        except TableNotFoundWarning:
+            if self.spec.report_meta.created_by_builder:
+                msg = _(
+                    'The database table backing your report does not exist yet. '
+                    'Please wait while the report is populated.'
+                )
+            else:
+                msg = _(
+                    'The database table backing your report does not exist yet. '
+                    'You must rebuild the data source before viewing the report.'
+                )
+            return self.render_json_response({'warning': msg})
+
+    def _render_datatable_page_bootstrap5(self, params):
```

Comment:
what is the difference between this and the rendering in `_get_ajax_bootstrap3`.
They look same but I could not spot a difference on quick read

Reply:
With Bootstrap 5, we are using DataTables 1.10. [Its parameter names have changed](https://datatables.net/manual/server-side).

You can find similar blocks of code for DataTables 1.10 in [PaginatedReportMixin](https://github.com/dimagi/commcare-hq/blob/1346a3d261e52956140326f6dbede889887689a8/corehq/apps/reports/generic.py#L1269) and [ApplicationStatusReport](https://github.com/dimagi/commcare-hq/blob/88c82969fc67c1338848af8122cd670b63d65239/corehq/apps/reports/standard/deployments.py#L172).